src/app/components/sandbox-controller/sandbox-controller.component.ts

```diff
@@ -38,7 +38,14 @@ export class SandboxControllerComponent implements OnInit {
 
   sources$ = this._sourcesSubject$.asObservable().pipe(distinctUntilChanged());
   output$ = this._outputSubject$.asObservable().pipe(distinctUntilChanged());
-  links$ = this._linksSubject$.asObservable();
+  links$ = this._linksSubject$.asObservable().pipe(
+    mergeMap((links) => this._hashedExampleSvc.getUrlTree(this.code$, this.sources$).pipe(
+      map((urlTree) => [
+        { label: 'Current example', url: this._router.serializeUrl(urlTree) },
```

Comment:
@lgpage let me know if you'd prefer the link to be a share button or something like that ;)

Reply:
@peterjokumsen yeah, I think I would prefer a button.
Maybe a share icon next to the toggle theme switch?

Reply:
A button that copies the url to the clipboard and replaces the current url?

Reply:
Will give that a go 👍🏼